src/data/faq.json

```diff
@@ -850,7 +943,7 @@
                                 "anchor": "admission_policy",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated on March 30, 2022:</b> Starting with version 2.18, the Corona-Warn-App will also show you the 2G+ status proof for 'freshly' vaccinated, 'freshly' recovered, and 'boostered' users if the corresponding COVID digital certificates are stored in the app and also have been issued for the same name as well as date of birth.",
+                                    "<b>Update on 28. September 2022: </b> <br> <span style='color:red;font-size: 24px;font-weight: bold;'>!</span><b>The status proof (3G, 3G+, 2G, 2G+) is currently not relevant and therefore not shown by the app</b><hr><b>Updated on March 30, 2022:</b> Starting with version 2.18, the Corona-Warn-App will also show you the 2G+ status proof for 'freshly' vaccinated, 'freshly' recovered, and 'boostered' users if the corresponding COVID digital certificates are stored in the app and also have been issued for the same name as well as date of birth.",

```

Comment:
- Since the above comment was submitted after this PR was merged, I transferred the information into a bug report https://github.com/corona-warn-app/cwa-website/issues/3111 just in case there is capacity to fix the problem.